tests/scripts/perf-prep.sh

```diff
@@ -64,19 +64,17 @@ python3.5 --version
 python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
 
 if [ $throughput -eq 1 ]; then
-    # Clone corefx
-    if [ -d "_" ]; then
-        rm -r -f _
+    # Download throughput benchmarks
+    if [ -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
+        rm -r -f Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
```

Comment:
Right now we don't have architecture parameterized in this script. Additionally, I *think* that we could remove the architecture all together, and only have one package for all of the TP assemblies, but I haven't tested that.

All that to say: yes, this was intentional, but if you think that I should parameterize it, I will.